gluster-exporter/metric_volume.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	log "github.com/sirupsen/logrus"
+)
+
+var (
+	volumeHealLabels = []MetricLabel{
+		{
+			Name: "volume",
+			Help: "Volume Name",
+		},
+		{
+			Name: "brick_path",
+			Help: "Brick Path",
+		},
+		{
+			Name: "host",
+			Help: "Hostname",
+		},
+	}
+
+	glusterVolumeHealCount = newPrometheusGaugeVec(Metric{
+		Namespace: "gluster",
+		Name:      "volume_heal_count",
```

Comment:
Is this heal pending count?

Reply:
yes. pending heal cnt.

Reply:
Ok. In that case should we rename the metric as `volume_pending_heal_count` ?

Reply:
ok